Overview.bs

```diff
@@ -346,21 +348,21 @@ For a PatchRequest object to be well formed:
 
 <table>
   <tr><th>ID</th><th>Field Name</th><th>Value Type</th></tr>
-
-  <tr><td>0</td><td>patch_format</td><td>Integer</td></tr>  
+  <tr><td>?</td><td>protocol_version</td><td>Integer ?????????????</td>
```

Comment:
sgtm on adding protocol version here. Can you assign this ID 0, bump the other id's by one and set type to Integer.